kaocha-runner.el

```diff
@@ -46,6 +46,28 @@
   :group 'kaocha-runner
   :type 'string)
 
+(defcustom kaocha-runner-ongoing-tests-win-min-height
+  12
+  "The minimum height in lines of the output window when tests are taking long to run.
+This is to show the ongoing progress from kaocha."
+  :group 'kaocha-runner
+  :type 'integer
+  :package-version '(kaocha-runner . "0.2.0"))
+
+(defcustom kaocha-runner-failure-win-min-height
+  4
+  "The minimum height in lines of the output window when there are failing tests."
+  :group 'kaocha-runner
+  :type 'integer
+  :package-version '(kaocha-runner . "0.2.0"))
+
+(defcustom kaocha-runner-details-win-max-height
+  16
+  "The maximum height in lines of the details window."
```

Comment:
The `details window` and the `output window` are the same window. Let's settle on a single name for it. The code calls it the `out-buffer`, so maybe output window is best?

Reply:
Done, do you want me to squash the commit?